.env

```diff
@@ -1 +1,5 @@
+GEO_API_KEY=pk.6845bc4d68078a6f65175f424e4f5214
+WEATHER_API_KEY=03b2bc2cde884978af331be2239ee12f
+TRAIL_API_KEY = 200970711-94e1fdc342eea8e6d3409a22b40aa126
+
```

Comment:
Hey that .env still got pushed up and once you start involving keys being pushed up, you want to be extra careful!